WordPress/src/main/java/org/wordpress/android/ui/sitecreation/theme/HomePagePickerFragment.kt

```diff
@@ -0,0 +1,119 @@
+package org.wordpress.android.ui.sitecreation.theme
+
+import android.content.Context
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.ViewModelProviders
+import androidx.recyclerview.widget.GridLayoutManager
+import com.google.android.material.appbar.AppBarLayout
+import kotlinx.android.synthetic.main.home_page_picker_bottom_toolbar.*
+import kotlinx.android.synthetic.main.home_page_picker_fragment.appBarLayout
+import kotlinx.android.synthetic.main.home_page_picker_fragment.layoutsRecyclerView
+import kotlinx.android.synthetic.main.home_page_picker_titlebar.*
+import kotlinx.android.synthetic.main.modal_layout_picker_subtitle_row.*
+import kotlinx.android.synthetic.main.modal_layout_picker_title_row.*
+import org.wordpress.android.R
+import org.wordpress.android.WordPress
+import org.wordpress.android.ui.sitecreation.theme.HomePagePickerViewModel.UiState
+import org.wordpress.android.util.AniUtils
+import org.wordpress.android.util.AniUtils.Duration
+import org.wordpress.android.util.DisplayUtils
+import org.wordpress.android.util.setVisible
+import javax.inject.Inject
+
+private const val NUM_COLUMNS = 2
+
+/**
+ * Implements the Home Page Picker UI
+ */
+class HomePagePickerFragment : Fragment() {
```

Comment:
Please ignore if I've missed some context on this: I wonder if we should subclass from `SiteCreationBaseFormFragment`? It seems that class has the help menu item with an abstract `onHelp` method we could implement here. Is there something that makes this incompatible / inconvenient for re-use? Wdyt?

Reply:
I agree with this @mkevins. I started implementing this way but since there are a lot of hook points with the existing UI (and refactoring in the existing code that need to be done) I decided to keep this PR small (under 500 lines). I will eventually need to handle this [in a next issue](https://github.com/wordpress-mobile/gutenberg-mobile/issues/2721) that seemed more targeted to this. Wdyt?

Reply:
That makes sense. Handling in the next PR is fine, and your explanation is sensible.